pkg/util/util.go

```diff
@@ -404,24 +405,25 @@ func chooseHostInterfaceNativeGo() (net.IP, error) {
 				return nil, err
 			}
 			if len(addrs) > 0 {
-				// This interface should suffice.
-				break
+				for _, addr := range addrs {
+					if addrIP, _, err := net.ParseCIDR(addr.String()); err == nil {
+						if addrIP.To4() != nil {
```

Comment:
minor nit: Any reason not to collapse the not nil check and the assignment into a single if statement? i.e. single call to To4().